### PR TITLE
Update polkadot api to 10.13.1

### PR DESCRIPTION
**Pull request by atodorov**
# Description of proposed changes

<describe what this PR is about and why we want it>

---

Practical tips for PR review & merge:

- [ ] All GitHub Actions report PASS
- [ ] Newly added code/functions have unit tests
  - [ ] Coverage tools report all newly added lines as covered
  - [ ] The positive scenario is exercised
  - [ ] Negative scenarios are exercised, e.g. assert on all possible errors
  - [ ] Assert on events triggered if applicable
  - [ ] Assert on changes made to storage if applicable
- [ ] Modified behavior/functions - try to make sure above test items are covered
- [ ] Integration tests are added if applicable/needed

### CODE REVIEW

File: cli/src/lib/staking/era.ts [hunk 1]
```diff
@@ -1,24 +1,4 @@
...
-    const minBlockTime = new BN(api.consts.babe.expectedBlockTime.toString());
-
-    const timeTillTargetEra = eraNumber
-        .sub(currentEra.add(new BN(1)))
-        .mul(slotsPerEra)
-        .add(blocksTillNextEra)
-        .mul(minBlockTime);
-
-    return timeTillTargetEra;
-}
```
Comment (DylanVerstraete) [inline cli/src/lib/staking/era.ts:21]: why was this removed?

Comment (atodorov) [reply cli/src/lib/staking/era.ts:21]: Because it is unused.

**Review by beqaabu — APPROVED** [review]

**Review by DylanVerstraete — APPROVED** [review]
